examples/tensorflow/training.ipynb

```diff
@@ -0,0 +1,173 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Single Node Single GPU Training in TensorFlow\n",
+    "\n",
+    "In this sample notebook, you'll train Resnet50 architecture to identify different species of birds. We are only running a few epochs, to save time, but once you've got this working you'll have all the information you need to build and run bigger Tensorflow models on Saturn Cloud.\n",
+    "\n",
+    "This notebook was run on a T4-4XLarge instance in tests, and we recommend not going much smaller with your instance size."
+   ],
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-08-16T21:29:50.228313Z",
+     "iopub.status.busy": "2021-08-16T21:29:50.228073Z",
+     "iopub.status.idle": "2021-08-16T21:29:50.231558Z",
+     "shell.execute_reply": "2021-08-16T21:29:50.230865Z",
+     "shell.execute_reply.started": "2021-08-16T21:29:50.228252Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "import tensorflow as tf\n",
+    "import keras\n",
+    "import time"
+   ],
+   "outputs": [],
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-09-27T17:31:17.736129Z",
+     "iopub.status.busy": "2021-09-27T17:31:17.735844Z",
+     "iopub.status.idle": "2021-09-27T17:31:17.739915Z",
+     "shell.execute_reply": "2021-09-27T17:31:17.739404Z",
+     "shell.execute_reply.started": "2021-09-27T17:31:17.736108Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "import s3fs\n",
+    "\n",
+    "s3 = s3fs.S3FileSystem(anon=True)\n",
+    "_ = s3.get(\n",
+    "    rpath=\"s3://saturn-public-data/100-bird-species/100-bird-species/*/*/*.jpg\",\n",
+    "    lpath=\"dataset/birds/\",\n",
+    ")"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "The first time you run this job, you'll need to download the training and test data in the code chunk above. It'll take a few moments, but it shouldn't be too long. This small sample only includes 61 classes, but the original has 285. "
```

Comment:
I don't love this sentence: "It'll take a few moments, but it shouldn't be too long." I'd prefer to have an estimate. Something like:  "This will take roughly 2 minutes." 

The time should be fairly consistent I think because it's all happening within AWS and the instance size is the same.